ui/page/livestreamCreate/view.jsx

```diff
@@ -0,0 +1,30 @@
+// @flow
+import React from 'react';
+import LivestreamForm from 'component/publish/livestream/livestreamForm';
+import Page from 'component/page';
+import YrblWalletEmpty from 'component/yrblWalletEmpty';
+import Spinner from 'component/spinner';
+
+type Props = {
+  balance: number,
+  fetchingChannels: boolean,
+};
+
+function LivestreamCreatePage(props: Props) {
+  const { balance, fetchingChannels } = props;
+
+  return (
+    <Page className="uploadPage-wrapper" noFooter>
+      {balance < 0.01 && <YrblWalletEmpty />}
+      {balance >= 0.01 && fetchingChannels ? (
+        <div className="main--empty">
+          <Spinner />
+        </div>
+      ) : (
+        <LivestreamForm disabled={balance < 0.01} />
+      )}
+    </Page>
+  );
+}
+
+export default LivestreamCreatePage;
```

Comment:
:exclamation: After creating a livestream, the following button doesn't bring me to my dashboard -- it just closes the modal.

<img width="421" alt="image" src="https://user-images.githubusercontent.com/64950861/177088395-d7594d80-6135-45db-a36f-0fd89acbaa4e.png">

Reply:
We don't really have a livestream dashboard (yet). This button just links to the setup page in production. I'll rename it to "View my publications" and redirect it to the uploads page. At least until we actually have a dashboard.

Reply:
Never mind, redirecting to the setup page makes more sense. The user might need the keys.